internal/commands/build.go

```diff
@@ -149,8 +149,10 @@ func Build(logger logging.Logger, cfg config.Config, packClient PackClient) *cob
 				DockerHost:        flags.DockerHost,
 				PullPolicy:        pullPolicy,
 				ClearCache:        flags.ClearCache,
-				TrustBuilder:      trustBuilder,
-				Buildpacks:        buildpacks,
+				TrustBuilder: func(string) bool {
+					return trustBuilder
```

Comment:
I have *not* actually moved the check, all I do in the CLI is to use the `trustBuilder` boolean to trust/untrust the builder.